web/docs/migration-guides/migrate-from-0-16-to-0-17.md

```diff
@@ -104,7 +104,68 @@ It is possible that you were not using this function in your code.
 If you're instead using [the `<LoginForm>` component](../auth/ui.md#login-form),
 this change is already handled for you.
 
-### 2. Enjoy your updated Wasp app
+### Update your `tsconfig.json`
+To ensure your project works correctly with Wasp 0.17.0, you must also update your
+`tsconfig.json` file.
+
+If you haven't changed anything in your project's `tsconfig.json` file (this is
+the case for most users), just replace its contents with the new version shown
+below.
+
+If you have made changes to your `tsconfig.json` file, we recommend taking the
+new version of the file and reapplying them.
+
+Here's the new version `tsconfig.json`:
+```json title=tsconfig.json
```

Comment:
I seem to remember you can use ` ```jsonc ` so that comments are supported? Maybe you can try it

Reply:
Tried it, doesn't work. But `json` renders comments normally in docosaurus:
![image](https://github.com/user-attachments/assets/a7dc93b8-ebd3-43e7-9d0b-c9ff727d7576)